tinygrad/renderer/cstyle.py

```diff
@@ -268,9 +268,10 @@ def render_kernel(self, function_name, kernel, bufs, uops, prefix=None):
                     UnaryOps.EXP2: lambda x,dtype: f"hexp2({x})" if dtype in (dtypes.half, dtypes.bfloat16) else f"exp2({x})",}
 
 _nms = "xyzwabcdefghijkl"
-def _make_cuda_dtype(base_type, cnt, align):
-  vec, elems, header = f"{base_type}{cnt}", ', '.join(_nms[:cnt]), ', '.join([f"{base_type} {x}" for x in _nms[:cnt]])
-  return f"struct __align__({align}) {vec} {{ {base_type} {elems}; }}; __device__ {vec} make_{vec}({header}) {{ {vec} r={{{elems}}}; return r; }}"
+def _make_cuda_dtype(self, dtype):
```

Comment:
can you type-check this, also why self? This isn't a class method.

Reply:
you are right, fixed